digdag-server/src/main/java/io/digdag/server/rs/ProjectResource.java

```diff
@@ -157,6 +157,8 @@
     private static final Logger logger = LoggerFactory.getLogger(ProjectResource.class);
     private static final int ARCHIVE_TOTAL_SIZE_LIMIT = 2 * 1024 * 1024;
     private static final int ARCHIVE_FILE_SIZE_LIMIT = ARCHIVE_TOTAL_SIZE_LIMIT;
+    private static final int MAX_SESSIONS_PAGE_SIZE = 1000;
```

Comment:
ditto

Reply:
Thank you for your reviewing!
Yes, I think it makes sense. I will try modifying to make `max page size` configurable by SystemConfig.

Reply:
Thank you!